### PR TITLE
Add css content and implement in index.html

### PR DESCRIPTION
**Pull request by RickKollenbroich93**

### CODE REVIEW

**Review by dimas1976 — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
Class and id names should be in english. You are mixing px with em, stay consistent.

File: styles.css [hunk 1]
```diff
@@ -3,3 +3,48 @@
 *::after {
   box-sizing: border-box;
 }
+body {
+  margin: 0;
+  font-family: "Archivo", sans-serif;
+  color: black;
+  font-size: 1em;
```
Comment (dimas1976) [inline styles.css:10]:
```suggestion
  font-size: 1rem;
```

File: styles.css [hunk 2]
```diff
@@ -3,3 +3,48 @@
...
+}
+.links {
+  list-style-type: none;
+  display: flex;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+}
+.header {
+  background: white;
```
Comment (dimas1976) [inline styles.css:26]:
```suggestion
  background-color: white;
```

File: styles.css [hunk 3]
```diff
@@ -3,3 +3,48 @@
...
+  height: auto;
+}
+.banner {
+  display: grid;
+  background: white;
```
Comment (dimas1976) [inline styles.css:31]: background-color

File: styles.css [hunk 4]
```diff
@@ -3,3 +3,48 @@
...
+  grid-template-columns: repeat(auto-fit, minmax(500px, 1fr));
+}
+.navbar {
+  display: grid;
+  background: hotpink;
```
Comment (dimas1976) [inline styles.css:36]: background-color

**Review by dimas1976 — CHANGES_REQUESTED** [review]

**Pull request merged**